[2025-03-31T14:06:36.260-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-31T14:06:36.265-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exec_py.taskC_hw_py manual__2025-03-31T19:06:23.930638+00:00 [queued]>
[2025-03-31T14:06:36.268-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exec_py.taskC_hw_py manual__2025-03-31T19:06:23.930638+00:00 [queued]>
[2025-03-31T14:06:36.269-0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-31T14:06:36.405-0500] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): taskC_hw_py> on 2025-03-31 19:06:23.930638+00:00
[2025-03-31T14:06:36.407-0500] {standard_task_runner.py:72} INFO - Started process 461428 to run task
[2025-03-31T14:06:36.408-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'exec_py', 'taskC_hw_py', 'manual__2025-03-31T19:06:23.930638+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/exec_py.py', '--cfg-path', '/tmp/tmp5hz00cl9']
[2025-03-31T14:06:36.409-0500] {standard_task_runner.py:105} INFO - Job 50: Subtask taskC_hw_py
[2025-03-31T14:06:36.560-0500] {task_command.py:467} INFO - Running <TaskInstance: exec_py.taskC_hw_py manual__2025-03-31T19:06:23.930638+00:00 [running]> on host royal-07.cs.wisc.edu
[2025-03-31T14:06:36.933-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='cloud.user' AIRFLOW_CTX_DAG_ID='exec_py' AIRFLOW_CTX_TASK_ID='taskC_hw_py' AIRFLOW_CTX_EXECUTION_DATE='2025-03-31T19:06:23.930638+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-31T19:06:23.930638+00:00'
[2025-03-31T14:06:36.933-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-31T14:06:36.940-0500] {logging_mixin.py:190} INFO - hello C
[2025-03-31T14:06:36.940-0500] {python.py:240} INFO - Done. Returned value was: None
[2025-03-31T14:06:36.943-0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-31T14:06:36.943-0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=exec_py, task_id=taskC_hw_py, run_id=manual__2025-03-31T19:06:23.930638+00:00, execution_date=20250331T190623, start_date=20250331T190636, end_date=20250331T190636
[2025-03-31T14:06:37.383-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-31T14:06:37.394-0500] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-31T14:06:37.671-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
